pkg/server/singleprocess/service_job.go

```diff
@@ -686,7 +688,7 @@ func (s *Service) GetJobStream(
 	ws := memdb.NewWatchSet()
 	job, err := s.state(ctx).JobById(req.JobId, ws)
 	if err != nil {
-		return err
+		return hcerr.Externalize(log, err, "failed to get job with id %q", req.JobId)
```

Comment:
```suggestion
		return hcerr.Externalize(log, err, "failed to get job with id", "job_id", req.JobId)
```

Should this use the `"job_id", req.JobId` format?